external/XML_IO_Tools/xml_write.m

```diff
@@ -86,7 +86,7 @@
 
 %% Check Matlab Version
 v = ver('MATLAB');
-v = str2double(regexp(v.Version, '\d.\d','match','once'));
+v = str2double(regexp(v.Version, '\d+.\d','match','once'));
```

Comment:
Nice catch! For the sake of completeness, I suggest using `'\d+.\d+'`, so it also matches the two-digit subversions. It is not necessary for the code to work, though.